Source/MultipartFormData.swift

```diff
@@ -189,43 +189,22 @@ open class MultipartFormData {
     ///
     /// The body part data will be encoded using the following format:
     ///
-    /// - `Content-Disposition: form-data; name=#{name}; filename=#{generated filename}` (HTTP Header)
-    /// - `Content-Type: #{generated mimeType}` (HTTP Header)
+    /// - `Content-Disposition: form-data; name=#{name}; filename=#{filename}` (HTTP Header)
+    /// - `Content-Type: #{mimeType}` (HTTP Header)
     /// - Encoded file data
     /// - Multipart form boundary
     ///
-    /// The filename in the `Content-Disposition` HTTP header is generated from the last path component of the
-    /// `fileURL`. The `Content-Type` HTTP header MIME type is generated by mapping the `fileURL` extension to the
-    /// system associated MIME type.
-    ///
-    /// - parameter fileURL: The URL of the file whose content will be encoded into the multipart form data.
-    /// - parameter name:    The name to associate with the file content in the `Content-Disposition` HTTP header.
-    public func append(_ fileURL: URL, withName name: String) {
```

Comment:
Could you also collapse the `Data` methods in the same way you have for the `URL` methods?

Reply:
Sure, let me do that.

Reply:
Done.